### DISCUSSION TITLE
Allow Non admin users to view all users.

### DISCUSSION THREAD

**Discussion by bob-pixeldeluxe** [craftcms/cms#8955] [Ideas]
Hi, 

We have a problem one of our customers does not have acces to all the users that are registered because allot of our users are not assigned to an group.
It seems that non admin users cannot view all users they can just select user groups. Can this be added?

Thanks in advance

**Comment by brandonkelly** [comment]
Go to their user preferences, and make sure they have permission to administrate users for each of the user groups you want them to be able to access.

**Comment by bob-pixeldeluxe** [comment]
Hi Brandonkelly,

Thanks for the reaction, We do have all permissions checked for this user and also in other sites we have this problem as well.
![Screenshot 2020-11-11 at 09 19 20](https://user-images.githubusercontent.com/50947605/98787014-394e5680-23ff-11eb-8133-cd3f77289305.png)
